src/components/FeatureViews/AdminPortal/TestPanel.js

```diff
@@ -0,0 +1,121 @@
+import React, {useEffect, useMemo, useState} from "react";
+import {connect, useDispatch, useSelector} from "react-redux";
+import {bindActionCreators} from "redux";
+import * as adminActions from "../../../actions/adminActions";
+
+import ReactDOM from 'react-dom';
+
+import PanelManager from "./PanelManager";
+import Paper from "@material-ui/core/Paper";
+import TextField from "@material-ui/core/TextField";
+import Grid from "@material-ui/core/Grid";
+import {Button, Typography} from "@material-ui/core";
+
+function TestPanel() {
+    
+    const dispatch = useDispatch();
+    const api = useMemo(
+        () => ({
+            ...bindActionCreators(adminActions, dispatch),
+        }),
+        [dispatch]
+    );
+    
+    const fields = [
+        {
+            "name": "Category Name",
+            "col-width": 3,
+            
+        },
+        {
+            "name": "Description",
+            "col-width": 7,
+            "editable": "true"
+        }
+    ];
+
+    const [categoryName, setCategoryName] = useState('');
+    const [categoryDescription, setCategoryDescription] = useState('');
+    const [categoryList, setCategoryList] = useState([]);
+
+    const categories = useSelector(({Course}) => Course.CourseCategories);
+    const categoryStatus = useSelector(({RequestStatus}) => RequestStatus.category);
+
+    
+    const customUseEffects = [];
+
+    useEffect(()=>{
+        api.fetchCategories();
```

Comment:
we should figure out if we want PanelManager to do the fetching or we want our parent component to do the fetching. Right now, we're fetching categories both in the PanelManager and this TestPanel component. I think keeping it in TestPanel would be good enough. @NeilAllavarpu what do you think?